Composer/packages/client/src/recoilModel/dispatchers/setting.ts

```diff
@@ -151,7 +151,7 @@ export const settingsDispatcher = () => {
   );
 
   const setRuntimeField = useRecoilCallback(
-    ({ set }: CallbackInterface) => async (projectId: string, field: string, newValue: boolean) => {
+    ({ set }: CallbackInterface) => async (projectId: string, field: string, newValue: boolean | string) => {
```

Comment:
is this necessary? how can a value be boolean or string?

Reply:
this is because runtime field could be string or boolean. We can set the string or boolean property in `setRuntimeField`
![image](https://user-images.githubusercontent.com/5860999/106867552-b2accb00-6708-11eb-87dd-3777e14459d0.png)